src/pages/ProviderEServiceTemplateCreatePage/components/EServiceTemplateCreateStepGeneral/EServiceTemplateCreateStepGeneral.tsx

```diff
@@ -64,8 +64,8 @@ export const EServiceTemplateCreateStepGeneral: React.FC = () => {
 
   const defaultValues: EServiceTemplateCreateStepGeneralFormValues = {
     name: template?.eserviceTemplate.name ?? '',
-    eserviceDescription: template?.eserviceTemplate.eserviceDescription ?? '',
-    audienceDescription: template?.eserviceTemplate.audienceDescription ?? '',
+    eserviceDescription: template?.eserviceTemplate.description ?? '',
+    audienceDescription: template?.eserviceTemplate.intendedTarget ?? '',
```

Comment:
Can you also rename the form properties?
```suggestion
    description: template?.eserviceTemplate.description ?? '',
    intendedTarget: template?.eserviceTemplate.intendedTarget ?? '',
```